FlowCrypt/Core/Core.swift

```diff
@@ -122,6 +122,15 @@ actor Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
         return CoreRes.EncryptFile(encryptedFile: encrypted.data)
     }
 
+    // MARK: - Messages
+    public func encryptMsg(msg: SendableMsg, fmt: MsgFmt) async throws -> CoreRes.ComposeEmail {
+        let r = try await call("encryptMsg", jsonDict: [
+            "pubKeys": msg.pubKeys,
+            "msgPwd": msg.password
+        ], data: msg.text.data())
+        return CoreRes.ComposeEmail(mimeEncoded: r.data)
```

Comment:
I'd return `Data` (`r.data`) directly here. Since the result is not `mimeEncoded` in this case anyway.